test/ui-testing/package-notes.js

```diff
@@ -0,0 +1,15 @@
+const Pkg = require('./er-notes');
+
+const PKG = {
```

Comment:
I'm not sure that hardcoding the package name here is the best way forward. The issue as I see it is that you're assuming that we'll always be running these tests on a platform that ran our particular start-up scripts which create that K-Int test package. 

That isn't a safe assumption - note that `folio-testing` doesn't even do that since [it doesn't have a K-Int Test Package.](https://folio-testing.aws.indexdata.com/erm/eresources?query=K-Int&sort=name).

I really like that you split off the testing code into `er-notes`, so I think it makes sense to just pass in a boolean like `isPackage`. Then, `er-notes` can check the **Is package: Yes** filter checkbox if `isPackage === true` and the **Is package: No** filter checkbox if `isPackage === false`. Then you can just pick the first result after the network goes idle and be assured that it's a package/title.